### PR TITLE
Add category additional description

### PR DESCRIPTION
**Pull request by Hlavtox**
Theme PR for https://github.com/PrestaShop/PrestaShop/pull/27292

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]
Looks fine but waiting for the core PR to be merged

**Review by kpodemski — APPROVED** [review]

**Review by matthieu-rolland — DISMISSED** [review]
request change is to block the PR until it can be merged, feel free to cancel it when it's time

**Review by khouloudbelguith — COMMENTED** [review]
Hi @Hlavtox,

It is ok :heavy_check_mark: 
While testing this PR, to not have a conflict with the classic them, I renamed this theme to classic-theme in config/theme.yml and I changed the preview.png

I checked a category with

- empty products
- Category with pagination
- I tried to change the page
- I tried to use some filter & sort => ok

https://user-images.githubusercontent.com/16067358/161952131-801a3ecb-469d-45a5-9eca-7d4316cb97c3.mp4

to test this theme, I followed these steps:

1. git clone git@github.com:PrestaShop/classic-theme.git
2. cd classic-theme
3. git prc origin 18 develop
4. npm install && npm run build

Thanks!

**Pull request merged**
